Python/challenges/day_3.py

```diff
@@ -0,0 +1,43 @@
+def create_item_dict():
+    item_dict = {}
+    for i in range(26):
+        letter = chr(i + ord("a"))
+        item_dict[letter] = [0, i + 1]
+    for j in range(26):
+        letter = chr(j + ord("A"))
+        item_dict[letter] = [0, j + 1 + 26]
+    return item_dict
+        
+def get_common_item(comps: list):
```

Comment:
Good parameter `comps`. This solution works with any number of collections greater or equal to 2.